### PR TITLE
chore: temp disable usdc/usdc.e routes for arb, op and base

### PR DESCRIPTION
**Pull request by dohaki**
Temporarily disable all USDC/USDC.e/USDbC routes for Arbitrum, Optimism, and Base. The only available routes will be 

#### Swaps on origin chain:
- 137 USDC.e --> {1,324,59144} USDC

#### Native USDC as inputToken w/o swap
- 1 USDC --> {137} USDC
- 1 USDC --> {137,324, 59144} USDC.e
- 137 USDC --> {1} USDC
- 137 USDC --> {1,324,59144} USDC.e

#### Bridged USDC as inputToken w/o swap
- 324 USDC.e --> {137,59144} USDC.e
- 324 USDC.e --> {1,137} USDC
- 59144 USDC.e --> {137,324} USDC.e
- 59144 USDC.e --> {1,137} USDC

### CODE REVIEW

File: scripts/generate-routes.ts [hunk 1]
```diff
@@ -293,12 +286,17 @@ const enabledRoutes = {
...
               "ETH",
               // { inputTokenSymbol: "USDC", outputTokenSymbol: "USDC.e" },
-              "USDC.e",
               "USDT",
               "DAI",
               "WBTC",
             ],
-            swapTokens: [],
+            swapTokens: [
+              // {
```
Comment (james-a-morris) [inline scripts/generate-routes.ts:294]: Is this meant to be an empty array?

Comment (dohaki) [reply scripts/generate-routes.ts:294]: Yes it is. Every route from OP related to USDC/USDC.e should be disabled

**Review by james-a-morris — APPROVED** [review]

**Comment by james-a-morris** [comment]
Spot checked this in the preview link

**Review by mrice32 — APPROVED** [review]
LGTM!

**Review by nicholaspai — APPROVED** [review]

**Pull request merged**